scripts/triggers.coffee

```diff
@@ -0,0 +1,51 @@
+# Description:
+#   Triggers allow you to make the robot respond with the given string.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   trigger phrase with response - Make the robot respond with "response" when it hears "phrase"
```

Comment:
This sounds weird. I'd like to improve this but I'm not sure what to replace it with :confounded: